Makefile

```diff
@@ -302,6 +305,14 @@ run-proxy:
 run-build-vm: $(BUILD_VM) $(BIOS_IMG) $(DEVICETREE_DTB)
 	$(QEMU_SYSTEM) $(QEMU_OPTS) -drive format=qcow2,file=$<
 
+run-parallels-vm:
+	mkdir $(LIVE).hdd
+	qemu-img convert -O parallels $(LIVE).qcow2 $(LIVE).hdd/live.0.$(PARALLELS_UUID).hds
```

Comment:
What you really need to do here is create a separate makefile target called `$(LIVE).parallels`. E.g.:

```
$(LIVE).parallels/live.0.$(PARALLELS_UUID).hds: $(LIVE).raw
     mkdir -p $(dirname $@)
     qemu-img convert -O parallels -f raw $<.tmp
     qemu-img info -f parallels --output json $<.tmp | jq --raw-output '.["virtual-size"]' | xargs ./tools/parallels_disk.sh $(LIVE) $(PARALLELS_UUID)
     ....
     mv $<.tmp $<
```